apps/api/tests/utils/testUtils.ts

```diff
@@ -52,7 +52,7 @@ export const initTestMigration = async (db: PostgresDb) => {
 /** Sets Number of Db Records to Seed @ start of test run using addInitialApplications */
 export const numTestApplications = 20;
 /** Used to bypass pagination impacting test results */
-export const allRecordsPageSize = 2 * numTestApplications;
+export const allRecordsPageSize = 5 * numTestApplications;
```

Comment:
This number is just arbitrarily large so that we always get all records returned for pagination tests